state/modeluser.go

```diff
@@ -193,6 +193,19 @@ func (e *UserModel) LastConnection() (time.Time, error) {
 	return lastConnDoc.LastConnection, nil
 }
 
+// AllUserModels returns a list of all models with the user they belong to.
+func (st *State) AllUserModels() ([]*UserModel, error) {
```

Comment:
Should we just be using this function instead of having both "AllModels" and "AllUserModels"?

It looks like the intent might be to suppress the "controller" model, is that true? or is it just annotated with the owner as well?